### PR TITLE
Added script to download OCM CLI

### PR DESCRIPTION
**Pull request by tirthct**

### CODE REVIEW

**Comment by cristianoveiga** [comment]
Let's make it possible to run this script via a Make file command.

**Comment by cristianoveiga** [comment]
Running the script returns:

mkdir: cannot create directory ‘tests/ocm’: No such file or directory

File: ensure_ocm_cli.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -eu
+ocmDownloads=$(curl -s https://api.github.com/repos/openshift-online/ocm-cli/releases/latest | grep "browser_download_url.*" | cut -d : -f 2,3 | tr -d \")
+osName=$(uname)
+osName="${osName,,}"
+hardware=$(uname -m)
+amd64="x86_64"
+if [[ "$hardware" = "$amd64" ]]; then
+    hardware="amd64"
+fi
```
Comment (cristianoveiga) [inline ensure_ocm_cli.sh:10]: Maybe add a comment to make it clear which OS and hardwares are currently supported.

File: ensure_ocm_cli.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```
Comment (cristianoveiga) [inline ensure_ocm_cli.sh:1]: Add a comment  briefly explaining what this script does and why it's needed.

Comment (tirthct) [reply ensure_ocm_cli.sh:1]: Added

**Comment by tirthct** [comment]
> Let's make it possible to run this script via a Make file command.

Added ensureOCM command in makefile

**Comment by cristianoveiga** [comment]
While running the downloaded ocm cli I am getting zsh: permission denied: ./ocm

Maybe this will cause issues in the future (during the tests).

Should we change the script to add the execute permission as well?

File: ensure_ocm_cli.sh [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+# This script downloads latest binaries of OCM CLI and stores them under tests/ocm for managing OCM CLI specific test dependencies
+set -eu
+ocmDownloads=$(curl -s https://api.github.com/repos/openshift-online/ocm-cli/releases/latest | grep "browser_download_url.*" | cut -d : -f 2,3 | tr -d \")
+# currently supported OS : darwin-amd64, linux-amd64, linux-arm64, linux-ppc64le, linux-s390, windows-amd64
```
Comment (cristianoveiga) [inline ensure_ocm_cli.sh:5]: Does it really work for windows?

**Comment by tirthct** [comment]
> While running the downloaded ocm cli I am getting zsh: permission denied: ./ocm
> 
> Maybe this will cause issues in the future (during the tests).
> 
> Should we change the script to add the execute permission as well?

Added execute permission

**Comment by cristianoveiga** [comment]
LGTM, but it would be good to print out some statements so that users know what's going on during the execution of the script.

**Comment by tirthct** [comment]
> LGTM, but it would be good to print out some messages so that users know what's going on during the execution of the script.

Added messages to the script

**Pull request merged**